src/main/webapp/util/ajax.js

```diff
@@ -440,3 +440,9 @@ export const ProjectMigration = {
     return axios.delete(UrlConstants.submissionsUrl + '?submissionId=' + submissionId);
   }
 };
+
+export const IssueList = {
```

Comment:
IssueList doesn't follow the naming pattern. Should not be named after an action but a resource: I suggest Issues , if doesn't fit on ConsentGroup or Project